src/lib/abacus/logger.ts

```diff
@@ -19,6 +21,20 @@ class AbacusLogger implements Omit<AbacusLoggingProtocol, '__doNotUseOrImplement
       console.error(`${tag}: ${message}`);
     }
   }
+
+  ddInfo(message: string, context: object) {
```

Comment:
@dydxprotocol/frontend this is how im planning on giving abacus the power to log info and error msgs to datadog.
it's a little hacky but until we're ready to merge their logs i figure this is good enough.